crates/matrix-sdk-crypto/src/store/integration_tests.rs

```diff
@@ -568,29 +568,22 @@ macro_rules! cryptostore_integration_tests {
             }
 
             #[async_test]
-            async fn test_load_inbound_group_sessions_by_room_id() {
-                let dir = "load_inbound_group_session_by_room_id";
+            async fn test_get_inbound_group_sessions_by_room_id() {
+                let dir = "get_inbound_group_session_by_room_id";
                 let (account, store) = get_loaded_store(dir).await;
                 assert_eq!(store.get_inbound_group_sessions().await.unwrap().len(), 0);
 
                 let room_id = &room_id!("!testing:localhost");
                 let (_, session_1) = account.create_group_session_pair_with_defaults(room_id).await;
                 let (_, session_2) = account.create_group_session_pair_with_defaults(room_id).await;
-                let (_, session_3) = account.create_group_session_pair_with_defaults(room_id).await;
-                let (_, session_4) = account.create_group_session_pair_with_defaults(room_id).await;
-                let (_, session_5) = account.create_group_session_pair_with_defaults(room_id).await;
-                let (_, session_6) = account.create_group_session_pair_with_defaults(room_id).await;
-                let (_, session_7) = account.create_group_session_pair_with_defaults(room_id).await;
 
+                let second_room_id = &room_id!("!other_room_testing:localhost");
+                let (_, session_3) = account.create_group_session_pair_with_defaults(second_room_id).await;
 
                 let mut sessions = vec![
-                    session_1.clone(),
-                    session_2.clone(),
-                    session_3.clone(),
-                    session_4.clone(),
-                    session_5.clone(),
-                    session_6.clone(),
-                    session_7.clone(),
+                    session_1,
+                    session_2,
+                    session_3
                 ];
 
                 for session in sessions.iter_mut() {
```

Comment:
can't this be simply:

```suggestion
                for session in sessions {
```

?

Reply:
I need to keep the sessions to compare later with loaded _sessions. while exporting them.

Reply:
Actually, my question should have been: what is the point of this loop? Why do we need to `export` and re-import the sessions? Can't we just remove this loop?